### ISSUE TITLE
Issues with 2FA: Unable to Login

### ISSUE THREAD

**Issue by jlambright** [rsms/fb-mac-messenger#369]
When attempting to login with an account, which has two factor authentication activated, a blank page is displayed after login. After refreshing from server, the "Login Verification" page briefly displays, but quickly returns to the blank page.

**Comment by corywatilo** [comment]
Yeah, sort of same here. Also seems to pop mobile Facebook open in the browser.

**Comment by kailan** [comment]
I had the same issue, but _"fixed"_ it by disabling 2FA, logging in, then re-enabling 2FA.

It's a bit hacky though so it'd be nice to have a real fix.

**Comment by willem-h** [comment]
I was able to login when the app sent me to Facebook in my browser to finish logging in. It pulled up the mobile version of Facebook in my desktop browser and when I checked my notifications it had a login attempt that I had to verify. Once I did that I refreshed the Mac App and it all worked nicely. There was a blank screen in between though.

**Comment by kde713** [comment]
I also use 2FA in Facebook Service, and I successfully signed in in this app by below steps.

```
1. Login on Mac Messenger App.
2. Redirect to Facebook Message
3. Click the button
4. Mobile facebook page will appear in your desktop browser. (or if login page appear, login in your desktop browser also)
5. Check the facebook notification. Maybe there are need to verify your activity notification.
6. click it > "Its' Me" > "Save browser"
7. Press "Command + R" in your mac app.
8. re-signin
9. done.
```

It works fine with above steps, but I think app needs some message page or some enhancement for 2FA users.
